setup.py

```diff
@@ -18,6 +18,7 @@
     'Fabric3>=1.10.2,<1.11',
     # can remove once requests bumps its version requirement
     # https://github.com/requests/requests/issues/4681
+    'gitpython~=3.1',
```

Comment:
How do we go about updating the requirements.txt file?

Reply:
```sh
make requirements
```